templates/style.php

```diff
@@ -279,6 +279,11 @@
 .amp-wp-article-featured-image {
 	margin: 0 0 1em;
 }
+.amp-wp-article-featured-image img {
```

Comment:
When native image is not enabled, this will get converted into `.amp-wp-article-featured-image amp-img`. To avoid duplicate styles being applied, should this do:

```suggestion
.amp-wp-article-featured-image img:not(amp-img) {
```